stactools_pipelines/pipelines/noaa_hrrr/config.yaml

```diff
@@ -0,0 +1,6 @@
+---
+  id: "noaa_hrrr"
+  compute: "awslambda"
+  secret_arn: "arn:aws:secretsmanager:eu-central-1:751953039297:secret:asdi-auth-stack-alukach/asdi-workflows-6awSos"
```

Comment:
should this be using 
>"arn:aws:secretsmanager:eu-central-1:751953039297:secret:asdi-auth-stack-dev/asdi-workflows-fSzAw7"

or ?

>"arn:aws:secretsmanager:us-west-2:751953039297:secret:asdi-auth-stack-dev/asdi-workflows-fSzAw7"

Reply:
I meant that we should use the "asdi-auth-stack-dev" secret with this ingestor instead of the "asdi-auth-stack-alukach" secret